Components/login/login.css

```diff
@@ -0,0 +1,41 @@
+.full-width {
+  width: 100vw;
+  height: 100vh;
+  box-sizing: border-box;
+  position: relative;
+}
+
+.back-to-home {
+  position: absolute;
+  top: 50px;
+  right: 50px;
+}
+
+.login-title {
+  margin: 2rem 0;
+}
+
+.login-container {
+  width: 500px;
+  min-height: 500px;
+  border: 1px solid var(--orange-500);
+}
+
+.input {
+  width: 100%;
+  padding: 1rem 2rem;
+  margin: 0;
+  box-sizing: border-box !important;
```

Comment:
You can use 
* {
  box-sizing : border-box;
 }
 If you want it for all the elements

Reply:
ok sure will correct it